_data/changelogs/component-form.yml

```
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Resolved `usa-fieldset` and `usa-legend` margin conflict in Chrome and Safari.
+    summaryAdditional: Now, `usa-fieldset` appropriately respects the margin set by child `usa-legend` elements. Confirm that your fieldset margins appear as expected. 
+    isBreaking: true
+    affectsStyles: true
+    githubPr: 5344
+    versionUswds: N.N.N
+    versionUswds: 3.6.0
```

Looks like the wrong line was deleted https://github.com/uswds/uswds-site/pull/2192/commits/cb15ee0020afbb9f0a4ab7f13fe7c929d292a03b#diff-c3cf1cf26267d864c6588f93dbd8ef72896fddf1c44e3f7b622e9b4b30225274L13
```suggestion
    githubRepo: uswds
    versionUswds: N.N.N
```

---

🤦‍♂️ 

I've been trying out the VSCode GH comments and looks like I haven't quite gotten them down yet! Thanks for catching this!

---

Resolved in [4cfa499](https://github.com/uswds/uswds-site/pull/2192/commits/4cfa49958d573dd3cf0c01a8b496985b4def0f6e)